Only output single currency breadcrumbs when rendering the settings page

luizreis opened the pull request:
Fixes #2282

#### Changes proposed in this Pull Request

The `save` method calls `get_settings`, causing the breadcrumbs to be rendered also when submitting the single currency settings form.

This PR adds a check to prevent rendering the breadcrumbs when not rendering the settings page, avoiding the warning and extra breadcrumbs at the top of the page.

<!--
Title: A descriptive, yet concise, title.
-->

<!--
Description: Write a brief summary about this PR. As you compose your summary, consider each of these questions and address them if appropriate. Why is this change needed? What does this change do? Were there other solutions you considered? Why did you choose to pursue this solution? Describe any trade-offs you might have had to make.
-->

<!--
Questions for PR author:
- How can this code break?
- What are we doing to make sure this code doesn't break?
-->

<!--
Images or gifs: Include before and after screenshots or gifs/videos when it makes sense.
-->

#### Testing instructions

<!--
Testing instructions: How should this be tested and how can a reviewer test the end-user functionality? Are there known issues that you plan to address in a future PR? Are there any side effects that readers should be aware of?
-->

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

* Navigate to the settings page for a single currency
* Make sure the breadcrumbs show up before the fields
* Update the form and submit it
* Make sure no breadcrumbs or warnings show up at the top of the page
* Make sure the breadcrumbs are displayed before the setting fields

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

<!--
Please read P7bbVw-3ww-p2 before opening the PR, assign the correct status to it, __and make sure that the related issue uses the Has PR status!__.
-->

tpaksu reviewed (approved):
I'm not sure if this needs a refactor to separate the output from `get_settings()` but I've tested it and LGTM. You can 🚢 it.

luizreis commented:
> I'm not sure if this needs a refactor to separate the output from `get_settings()` but I've tested it and LGTM. You can 🚢 it.

Good call @tpaksu. I've refactored it to use the `woocommerce_settings_wcpay_multi_currency` hook in https://github.com/Automattic/woocommerce-payments/pull/2284/commits/9dc2df87e9c568101317b3a459e8c95d2fc6598b and added some tests around that. Would you mind taking a look at it?

tpaksu commented:
Yes, it works. And looks better. Thanks for refactoring it :) Let's 🚢 it!
